doc/README

```diff
@@ -35,10 +35,6 @@ To get CUTEst working, here is what you need to do:
 
     To obtain problems from the CUTEr/st and related test sets, see
 
-       http://tracsvn.mathappl.polymtl.ca/trac/cuter/wiki/ProblemCollections
-
-    or
-
        http://www.cuter.rl.ac.uk/Problems/mastsif.shtml
```

Comment:
```suggestion
       https://bitbucket.org/optrove/sif/src/master
```
That's where the up-to-date problems now live.

Reply:
Even better, works well